app/src/main/java/com/besome/sketch/editor/PropertyActivity.java

```diff
@@ -0,0 +1,358 @@
+        this.t.setAdapter(new a());
+        this.m = (CustomScrollView) findViewById(2131231692);
+        this.l = (LinearLayout) findViewById(2131230932);
+        if (bundle != null) {
+            this.q = bundle.getString("sc_id");
+            this.n = (ProjectFileBean) bundle.getParcelable("project_file");
+            this.o = (ViewBean) bundle.getParcelable("bean");
+        } else {
+            this.q = getIntent().getStringExtra("sc_id");
+            this.n = (ProjectFileBean) getIntent().getParcelableExtra("project_file");
+            this.o = (ViewBean) getIntent().getParcelableExtra("bean");
+        }
+        d().a(this.o.id.charAt(0) == '_' ? this.o.id.substring(1) : this.o.id);
+        this.r = (LinearLayout) findViewById(2131231310);
+        this.r.setVisibility(8);
+        this.j.h();
+    }
+
+    public boolean onCreateOptionsMenu(Menu menu) {
+        getMenuInflater().inflate(2131492885, menu);
+        return true;
+    }
+
+    @Override
+    // androidx.appcompat.app.AppCompatActivity, androidx.fragment.app.FragmentActivity, com.besome.sketch.lib.base.BaseAppCompatActivity
+    public void onDestroy() {
+        super.onDestroy();
+    }
+
+    public boolean onOptionsItemSelected(MenuItem menuItem) {
+        if (menuItem.getItemId() == 2131231496) {
+            p();
+        }
+        return super.onOptionsItemSelected(menuItem);
+    }
+
+    @Override // androidx.appcompat.app.AppCompatActivity
+    public void onPostCreate(Bundle bundle) {
+        super.onPostCreate(bundle);
+        this.u = new ViewPropertyItems(this);
+        this.u.setOrientation(1);
+        l();
+    }
+
+    @Override
+    // androidx.fragment.app.FragmentActivity, com.besome.sketch.lib.base.BaseAppCompatActivity
+    public void onResume() {
+        super.onResume();
+        if (!super.j()) {
+            finish();
+        }
+        if (this.j.h()) {
+            n();
+        }
+        this.d.setScreenName(PropertyActivity.class.getSimpleName().toString());
+        this.d.send(new HitBuilders.ScreenViewBuilder().build());
+    }
+
+    @Override
+    // androidx.core.app.ComponentActivity, androidx.appcompat.app.AppCompatActivity, androidx.fragment.app.FragmentActivity
+    public void onSaveInstanceState(Bundle bundle) {
+        bundle.putString("sc_id", this.q);
+        bundle.putParcelable("project_file", this.n);
+        bundle.putParcelable("bean", this.o);
+        super.onSaveInstanceState(bundle);
+    }
+
+    public void p() {
+        Intent intent = new Intent(getApplicationContext(), ManageImageActivity.class);
+        intent.setFlags(536870912);
+        intent.putExtra("sc_id", this.q);
+        startActivityForResult(intent, 209);
+    }
+
+    class a extends RecyclerView.a<PropertyActivity.a.View$OnClickListenerC0029a> {
+
+        public a() {
+        }
+
+        /* renamed from: a */
+        public void b(View$OnClickListenerC0029a aVar, int i) {
```

Comment:
hmmmmmm, it doesn't seem like that's gonna work at runtime

Reply:
it's just conflicting with another a